build/ci-build.yml

```diff
@@ -0,0 +1,107 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - src/*
+
+pr:
+  paths:
+    include:
+      - src/*
+      - build/ci-build.yml
+
+variables:
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.105'
+  - name: 'Project'
+    value: 'Arcus.Template'
+  - name: 'Arcus_Api_BaseUrl'
+    value: 'http://localhost:5000/api/v1'
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
```

Comment:
Is it correct to assume we are only building the codebase here to run the tests given Dockerfile should build the required sources?

If so, I'd suggest to have only one stage to build the container, spin it up and run dotnet test on the codebase which will first run it anyway.

Would that make sense?

Reply:
Ah, yes. Now you remind, it was indeed something I saw in the logs.

Will make the YAML bit simpler. Thanks!

Reply:
My pleasure!

Reply:
Outdated: Dockerfile doesn't build this anymore